erasure/src/reed_solomon.rs

```diff
@@ -0,0 +1,87 @@
+use crate::error::Result;
+
+use reed_solomon_erasure::ReedSolomon;
+
+pub const fn align() -> usize {
+    1
```

Comment:
really?

Reply:
There don't seem to be any alignment requirements using the Rust library (or at least no exposed ones) so I just put 1